Detectors/TPC/workflow/src/tpc-reco-workflow.cxx

```diff
@@ -137,7 +138,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   std::vector<int> laneConfiguration = tpcSectors; // Currently just a copy of the tpcSectors, why?
   auto nLanes = cfgc.options().get<int>("tpc-lanes");
   auto inputType = cfgc.options().get<std::string>("input-type");
-
+  auto requireCTPLumi = cfgc.options().get<bool>("require-ctp-lumi");
```

Comment:
Shouldn't this be used in the `getWorkflow` call below?

Reply:
Thanks for spotting, @wiechula , indeed, the option was checked but not passed to the getWorkflow